### ISSUE TITLE
Fase 1: Preparación del Dataset Visual (CIFAR-10)

### ISSUE THREAD

**Issue by Serdan1** [Serdan1/cnn-cifar10#1]
escripción:
En esta fase se preparará el dataset CIFAR-10 para ser utilizado en el entrenamiento de la red neuronal convolucional (CNN).
El objetivo es explorar visualmente las imágenes, comprender su estructura y realizar el preprocesamiento necesario para que el modelo pueda aprender correctamente.

🎯 Objetivos

Cargar el dataset CIFAR-10 desde tensorflow.keras.datasets.

Visualizar ejemplos representativos de las 10 categorías.

Normalizar los valores de los píxeles al rango [0, 1].

Convertir las etiquetas de clase al formato one-hot encoding.

Mantener la estructura 2D de las imágenes (no aplanar).

🧠 Tareas

 Importar y cargar el dataset CIFAR-10 (x_train, y_train, x_test, y_test).

 Crear una función para visualizar al menos una imagen de cada clase con su etiqueta correspondiente.

 Normalizar los valores de los píxeles dividiendo entre 255.

 Convertir las etiquetas a formato one-hot encoding con to_categorical.

 Verificar las dimensiones finales de x_train y y_train.

 Documentar los resultados en el Notebook (Markdown + gráficos).

🧾 Criterios de finalización

El dataset se carga correctamente sin errores.

Se muestran imágenes representativas de todas las categorías.

Los valores de píxeles están normalizados (entre 0 y 1).

Las etiquetas están correctamente codificadas en formato one-hot.

Las formas (shape) de los datos son las esperadas:

x_train.shape = (50000, 32, 32, 3)

y_train.shape = (50000, 10)
